src/udev/udev-builtin-path_id.c

```diff
@@ -397,7 +397,7 @@ static struct udev_device *handle_scsi_default(struct udev_device *parent, char
         return hostdev;
 }
 
-static struct udev_device *handle_scsi_hyperv(struct udev_device *parent, char **path) {
+static struct udev_device *handle_scsi_hyperv(struct udev_device *parent, char **path, unsigned int guid_str_len) {
         struct udev_device *hostdev;
         struct udev_device *vmbusdev;
         const char *guid_str;
```

Comment:
I think `char guid[38]` below seems to be updated to `char guid[39]`.

Reply:
Thank you. I will fix it.